content/community/teams.md

```diff
@@ -5,53 +5,55 @@ next: /community/rules/
 
 # Teams and Organization
 
-_This page is a stub._
-
 ## Administrators
 
 Administrators are people who created, maintain, and are responsible for Codewars overall. They handle the most critical issues, like:
-- severe cases of violations of rules,
-- user accounts,
-- bugs,
-- ...?
+- Development and maintenance of Codewars platform,
+- Fixing platform bugs related to Codewars functions, user interface, code runner,
+- Managing user accounts,
+- Handling severe cases of violations of rules.
 
-Cuttently, users with status of Administrator are:
+Cuttently, active users with status of Administrator are:
 
-- [kazk](link to Codawars profile?)
-- ... others?
+- [kazk](https://www.codewars.com/users/kazk)
 
 They can be recognized by symbol of a crown (👑) displayed by their user name.
 
 ## Moderators
 
-Moderators are small team of users selected by Administrators to support them in handling some tasks related to community and Codewars content. Moderators have tools and privileges which help with maintenance of content and with handling of some community issues. Moderators can:
-- moderate discourses,
-- validate or flag solutions,
-- identify cheaters and abusers,
-- revoke privileges,
-- maintain documentation.
+Moderators are small team of users selected by Administrators to support them in handling some tasks related to community and Codewars content. Moderators have tools and privileges which help with maintenance of content and with handling of some community issues. Moderators have access to tools which allow them to:
+- Moderate discourses,
+- Manage submitted solutions: re-test them, flag as cheats, etc.
+- Identify cheaters, plagiarists, and users who do not play the game fair,
+- Handle abusers and revoke their privileges,
+- Maintain documentation.
```

Comment:
> - Maintain documentation.

This might need to be clarified. Anyone can contribute to the docs by opening PRs. Moderators are responsible for merging/rejecting them.